verticapy/machine_learning/model_selection/statistical_tests/tsa.py

```diff
@@ -297,6 +299,66 @@ def mkt(
     -------
     TableSample
         result of the test.
+
+    Examples
+    ---------
+
+    Let's try this test on a dummy dataset that has the
+    following elements:
+
+    - A value of interest
+    - Time-stamp data
+
+    Before we begin we can import the necessary libraries:
+
+    .. ipython:: python
+
+        import verticapy as vp
+
+    Now we can create the dummy dataset:
+
+    .. ipython:: python
+
+        vdf = vp.vDataFrame(
+            {
+                "X": [0, 1, 2, 3, 4],
+                "year": [1990, 1991, 1992, 1993, 1994],
+            }
+        )
+
+    To test its score, we can import the test function:
+
+    .. ipython:: python
+
+        from verticapy.stats import mkt
+
+    And simply apply it on the ``vDataFrame``:
+
+    .. ipython:: python
+
+        mkt(vdf, column = "X", ts = "year")
+
+    We can contrast the results with a dataset that
+    has barely any trend:
+
+    .. ipython:: python
+
+        vdf = vp.vDataFrame(
+            {
+                "X":[0, 1, 1, 1, 1],
+                "year": [1990, 1991, 1992, 1993, 1994],
+            }
+        )
+
```

Comment:
please draw the time series and add more elements. Visual content is always more impressive